packages/artillery-plugin-publish-metrics/lib/open-telemetry/index.js

```diff
@@ -395,6 +424,117 @@ class OTelReporter {
     }
   }
 
+  tracePerformanceFlow(performanceEntries, specName, userFlowFunc) {
+    // Set all timing events we are interested in and map their start and end to their respective property names in the performanceEntry object
+    // We can then use this map to set spans for each of these events if their start and end times are present
+    const timingEventsMap = {
+      redirect: { start: 'redirectStart', end: 'redirectEnd' },
+      fetch: { start: 'fetchStart', end: 'responseEnd' },
+      DNS_lookup: { start: 'domainLookupStart', end: 'domainLookupEnd' },
+      TCP_handshake: { start: 'connectStart', end: 'connectEnd' },
+      TLS_negotiation: { start: 'secureConnectionStart', end: 'requestStart' },
+      Request: { start: 'requestStart', end: 'responseStart' },
+      Response: { start: 'responseStart', end: 'responseEnd' },
+      'DOM.content_loaded.event': {
+        start: 'domContentLoadedEventStart',
+        end: 'domContentLoadedEventEnd'
+      },
+      'load.event': { start: 'loadEventStart', end: 'loadEventEnd' }
```

Comment:
We should have a more consistent naming convention here. I can see in the Honeycomb screenshot it looks a bit inconsistent. Perhaps:

- `redirect`
- `fetch`
- `dns_lookup`
- `tcp_handshake`
- `tls_negotiation`
- `request`
- `response`
- `dom_content_loaded` (do we need event?)
- `load` (do we need event?) 

Using `snake_case` should be better as anyway a lot of them get converted to that standard it seems (it seems Honeycomb/OTeL did it behind the scenes - probably due to Prometheus conventions).